MIT.md

```diff
@@ -1,4 +1,4 @@
-## Copyright 2021, [YOUR NAME]
+## Copyright 2021, [JUSTO RAGO, DANYS LINARES]
 
 ###### Please delete this line and the next one
```

Comment:
- [ ] kindly customize your license file to look professional by remove unnecessary content from it.